packages/jest-matchers/src/index.js

```diff
@@ -38,32 +38,32 @@ const {
 const GLOBAL_STATE = Symbol.for('$$jest-matchers-object');
 
 class JestAssertionError extends Error {
-  matcherResult: any
+  matcherResult: any;
 }
 
 if (!global[GLOBAL_STATE]) {
-  Object.defineProperty(
-    global,
-    GLOBAL_STATE,
-    {value: {
+  Object.defineProperty(global, GLOBAL_STATE, {
+    value: {
       matchers: Object.create(null),
       state: {
         assertionsExpected: null,
         assertionsMade: 0,
         suppressedErrors: [],
       },
-    }},
-  );
+    },
+  });
 }
 
 const expect: Expect = (actual: any): ExpectationObject => {
   const allMatchers = global[GLOBAL_STATE].matchers;
   const expectation = {not: {}};
   Object.keys(allMatchers).forEach(name => {
-    expectation[name] =
-      makeThrowingMatcher(allMatchers[name], false, actual);
-    expectation.not[name] =
-      makeThrowingMatcher(allMatchers[name], true, actual);
+    expectation[name] = makeThrowingMatcher(allMatchers[name], false, actual);
+    expectation.not[name] = makeThrowingMatcher(
+      allMatchers[name],
+      true,
+      actual,
+    );
```

Comment:
This is a great example of what I'd expect from prettier to do in such case:
input:
```js
expectation.not[name] =
  makeThrowingMatcher(allMatchers[name], true, actual);
```

first try:
```js
expectation.not[name] = makeThrowingMatcher(allMatchers[name], true, actual); // too long, move along
```

second try:
```js
expectation.not[name] =
  makeThrowingMatcher(allMatchers[name], true, actual); // ok!
```

third one if it still gets too long:
```js
expectation.not[name] = makeThrowingMatcher(
  allMatchers[name], 
  true, 
  actual,
  someLongerVariable
);
```